src/main/java/uk/gov/hmcts/reform/civil/callback/CaseEvent.java

```diff
@@ -365,7 +365,8 @@ public enum CaseEvent {
     GENERATE_DASHBOARD_NOTIFICATION_CUI(CAMUNDA),
     CREATE_DASHBOARD_NOTIFICATION_FOR_CLAIM_ISSUE_FOR_APPLICANT1(CAMUNDA),
     CREATE_DASHBOARD_NOTIFICATION_FOR_CLAIM_ISSUE_FOR_RESPONDENT1(CAMUNDA),
-    GENERATE_DASHBOARD_NOTIFICATION_CLAIM_ISSUE_HWF_CLAIMANT1(CAMUNDA);
+    GENERATE_DASHBOARD_NOTIFICATION_CLAIM_ISSUE_HWF_CLAIMANT1(CAMUNDA),
+    CREATE_CLAIMANT_DASHBOARD_NOTIFICATION_FOR_CLAIMANT_RESPONSE(CAMUNDA);
```

Comment:
should we use APPLICANT1 / RESPONDENT1 to avoid confusion claimant, defendant, applicant, respondent...??